client/layout/guided-tours/tours/editor-basics-tour.js

```diff
@@ -83,6 +83,30 @@ export const EditorBasicsTour = makeTour(
 				}
 			</p>
 			<ButtonRow>
+				<Next step="add-other" />
+				<Quit />
+			</ButtonRow>
+		</Step>
+		<Step
+			name="add-other"
+			arrow="top-left"
+			target=".editor-html-toolbar__button-insert-media, .mce-wpcom-insert-menu button"
+			placement="below"
+			style={ { marginLeft: '22px', zIndex: 'auto' } }
+		>
+			<p>
+				{
+					translate( 'The {{icon/}} lets you add other things, like a contact form. ' +
```

Comment:
Just a very minor, non-blocking nitpick here:
The indentation seems a little off to me, how do you feel about:

```jsx
translate( 
  'The {{icon/}} lets you add other things, like a contact form. ' +
  'If your site is on the Premium or Business plan, you can even add {{strong}}payment buttons{{/strong}}!', 
  {
    components: {
      ...
    }
  }
```
Rational: 
- keeps the text grouped together (without unrelated syntax sharing the same line. It's boundaries are easier to see at a glance.
- the { of the attributes object doesn't get lost at the end of the 2nd line of text and instead is more obvious.